src/bundler.ts

```diff
@@ -188,21 +186,40 @@ class Bundler {
         encodeString(script.document.parsedDocument.contents, true);
     dom5.removeAttribute(externalScript, 'src');
     dom5.setTextContent(externalScript, scriptContent);
+
+    return externalScript;
```

Comment:
It'd be good to have a comment that this returns the external script, and probably that it mutates some of its inputs (though this probably applies to lots of methods in the bundler...)